internal/client/api_account.go

```diff
@@ -0,0 +1,188 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// CreateAccount struct.
+type CreateAccount struct {
+	Name       string `json:"name"`
+	ID         string `json:"id"`
+	Password   string `json:"password"`
+	FullName   string `json:"fullName,omitempty"`
+	IsActive   bool   `json:"isActive,omitempty"`
+	IsAdmin    bool   `json:"isAdmin,omitempty"`
+	IsOrg      bool   `json:"isOrg,omitempty"`
+	SearchLDAP bool   `json:"searchLDAP,omitempty"`
+}
+
+// UpdateAccount struct.
+type UpdateAccount struct {
+	FullName string `json:"fullName,omitempty"`
+	IsActive bool   `json:"isActive,omitempty"`
+	IsAdmin  bool   `json:"isAdmin,omitempty"`
+}
+
+// ResponseAccount struct.
+type ResponseAccount struct {
+	Name         string `json:"name"`
+	ID           string `json:"id"`
+	FullName     string `json:"fullName,omitempty"`
+	IsActive     bool   `json:"isActive"`
+	IsAdmin      bool   `json:"isAdmin"`
+	IsOrg        bool   `json:"isOrg"`
+	IsImported   bool   `json:"isImported"`
+	OnDemand     bool   `json:"onDemand"`
+	OtpEnabled   bool   `json:"otpEnabled"`
+	MembersCount int    `json:"membersCount"`
+	TeamsCount   int    `json:"teamsCount"`
+}
+
+// Account filters enum.
+type AccountFilter string
+
+const (
+	Users                AccountFilter = "user"
+	Orgs                 AccountFilter = "orgs"
+	Admins               AccountFilter = "admins"
+	NonAdmins            AccountFilter = "non-admins"
+	ActiveUsers          AccountFilter = "active-users"
+	InactiveUsers        AccountFilter = "inactive-users"
+	URLTargetForAccounts               = "accounts"
+)
+
+// APIFormOfFilter is a string readable form of the AccountFilters enum.
+func (accF AccountFilter) APIFormOfFilter() string {
+	filters := [...]string{"users", "orgs", "admins", "non-admins", "active-users"}
```

Comment:
duplicate from the above constants. You should probably be reusing them instead of writing magic strings.

Reply:
the above constants are of type AccountFilter therefore not being able to use them as strings